systemd/fty-db-init.service.in

```diff
@@ -32,4 +32,3 @@ ExecStopPost=/bin/dash -c "/bin/systemctl stop -- $(/bin/systemctl show -p Wante
 
 [Install]
 WantedBy=bios.target
-Alias=bios-db-init.service
```

Comment:
I think this name should stay - some CI tests on services and REST API calls can still call the old name, and at least for direct `systemctl <action> bios-db-init` calls it does work.